sdk/eventhubs/microsoft-azure-eventhubs/pom.xml

```diff
@@ -36,7 +35,7 @@
     <dependency>
       <groupId>org.apache.qpid</groupId>
       <artifactId>proton-j</artifactId>
-      <version>0.33.8</version> <!-- {x-version-update;org.apache.qpid:proton-j;external_dependency} -->
+      <version>0.33.4-rcca-5308-1</version> <!-- {x-version-update;org.apache.qpid:proton-j;external_dependency} -->
```

Comment:
it seems weird but any reasons we are downgrading the version

Reply:
I was also wondering the same. ChrisE has made his changes and tested on top of `0.33.4` so i stick to that.